packages/radix-ui-themes/src/components/base-tab-list.css

```diff
@@ -0,0 +1,140 @@
+.rt-BaseTabList {
+  display: flex;
+  overflow-x: auto;
+  white-space: nowrap;
+
+  scrollbar-width: none;
+  &::-webkit-scrollbar {
+    display: none;
+  }
+}
+
+.rt-BaseTabListTrigger {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  flex-shrink: 0;
+  position: relative;
+  user-select: none;
+}
+
+.rt-BaseTabListTriggerInner,
+.rt-BaseTabListTriggerInnerHidden {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.rt-BaseTabListTriggerInner {
+  position: absolute;
+
+  :where(.rt-BaseTabListTrigger[data-state='inactive'], .rt-TabNavLink:not([data-active])) & {
+    letter-spacing: var(--tab-inactive-letter-spacing, var(--tabs-trigger-inactive-letter-spacing));
+    word-spacing: var(--tab-inactive-word-spacing, var(--tabs-trigger-inactive-word-spacing));
```

Comment:
I think we want to reverse the order of the variables here so that the new ones don't override any existing custom values

Reply:
Ah yeah good point, switching…